sdk/batch/arm-batch/src/models/applicationOperationsMappers.ts

```diff
@@ -1,19 +1,72 @@
 /*
  * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
was this change caused by the pre-commit hook that runs `prettier`? I would recommend to ignore any changes caused by that hook in this PR because this is auto-generated library and we should not edit it in anyway.

Reply:
How does the pre-commit hook usually skip the auto generated packages?

Reply:
It only skip files where their path matches the patterns set in this file: https://github.com/Azure/azure-sdk-for-js/blob/master/.prettierignore. It is easy to to detect auto-generated code and ignore it in track2 packages because they are always under `src/generated`. For track1, we do nothing at the moment. I think to fix this we need to hardcode the paths to packages to ignore or to packages to only work on.

Reply:
Actually, this is not due to pre-commit hook (I didn't install it when I checked in the first commit). This was generated by autorest.

Reply:
That makes sense because it is managed by `rush` and you do not need to use it.